pkg/resources/testdata/TestAcc_GrantApplicationRole/account_role/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "parent_account_role_name" {
+  type = string
+}
+
+variable "database_name" {
+  type = string
+}
+
+variable "schema_name" {
+  type = string
+}
+
+variable "application_name" {
+  type = string
+}
+
+variable "random_name" {
```

Comment:
it is used for stage and application package, let's be explicit by introducing stage_name and application_package_name instead

Reply:
Fixed.